data.json

```diff
@@ -1625,4 +1625,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
     "issueNumber": 853
   }
```

Comment:
Please add a comma after the brace, on line `1627` @srini047 .

Reply:
I have done the necessary changes @CodingSpecies . Please re-review my PR.